test/JDBC/input/BABEL-2934-vu-prepare.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE dbo.babel2934_1 (c TIME(7) NULL, d TIME(7) NULL)
```

Comment:
recommended object naming convention is filename_xxx, so it should be BABEL-2934-vu-prepare_1. This is easy to follow and consistent across the board.

Reply:
thanks, I'll make this change on the next revision